hypervisor/debug/profiling.c

```diff
@@ -871,7 +871,7 @@ int32_t profiling_vm_list_info(struct acrn_vm *vm, uint64_t addr)
 	vm_info_list.vm_list[vm_idx].num_vcpus = i;
 	vm_info_list.num_vms = 1;
 
-	for (j = 0U; j < CONFIG_MAX_VM_NUM; j++) {
+	for (j = 0U; j < CONFIG_MAX_VM_NUM-1; j++) {
```

Comment:
for (j = 1U; j < CONFIG_MAX_VM_NUM; j++) ?

Reply:
@dongyaoz, @mlim19  Actually, original logic is correct.  We loop through 0 to CONFIG_MAX_VM_NUM and populate vm info in profiling_vm_info_list. But size of profiling_vm_info_list is not sufficient causing out of bounds error, so increased it to CONFIG_MAX_VM_NUM+1 .